docs/mdbook/default.nix

```diff
@@ -204,9 +204,7 @@ with lib; let
             then "cp ${mkMDDoc opts.index.options} ${path}"
             else ''
               {
-              cat <<EOF
-              ${opts.index.moduleDoc}
-              EOF
+              cat ${pkgs.writeText "module-doc" opts.index.moduleDoc}
```

Comment:
Can you re-add the comment motivating this change please ?

Reply:
Sure thing